server/sockets.js

```diff
@@ -5,9 +5,10 @@ const createSocket = (listener) => {
   const io = socketio.listen(listener);
 
   io.use((socket, next) => {
-    if(socket.request.headers.cookie ){
+    if (socket.request.headers.cookie) {
```

Comment:
sorted... although interestingly the linter didn't catch this. We probs need better linter setup too...